test/which.js

```diff
@@ -69,5 +70,25 @@ test('Searching with -a flag returns an array with first item equals to the regu
   t.falsy(shell.error());
   t.truthy(resultForWhich);
   t.truthy(resultForWhichA);
-  t.is(resultForWhich.toString(), resultForWhichA[0].toString());
+  t.is(resultForWhich.toString(), resultForWhichA[0]);
+});
+
+test('None executable files does not appear in the result list', t => {
+  const commandName = 'node'; // Should be an existing command
+  const pathEnv = process.env.path || process.env.Path || process.env.PATH;
```

Comment:
This doesn't look right. This should be `process.env.PATH`.

Only all-caps `$PATH` works on unix. On Windows, `process.env.path` is totally the same as `process.env.PATH` (and all variations of capitalization of "path").